PowerEditor/src/Notepad_plus.cpp

```diff
@@ -5315,58 +5315,86 @@ void Notepad_plus::loadCommandlineParams(const TCHAR * commandLine, const CmdLin
 		return;
 	}
 
- 	LangType lt = pCmdParams->_langType;
-	int ln =  pCmdParams->_line2go;
-    int cn = pCmdParams->_column2go;
-    int cpos = pCmdParams->_pos2go;
+	LangType lt = pCmdParams->_langType;
+	int ln = pCmdParams->_line2go;
+	int cn = pCmdParams->_column2go;
+	int cpos = pCmdParams->_pos2go;

```

Comment:
Make your modification compact - don't reformat source code in your request. It makes code review more difficult.
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests